.github/workflows/tests-schedule.yml

```diff
@@ -36,7 +36,7 @@ jobs:
         run: pip install --no-build-isolation --editable .
 
       - name: Install all optional dataset requirements
-        run: pip install scipy pycocotools lmdb requests
+        run: pip install scipy pycocotools lmdb requests "git+https://github.com/pmeier/gdown@virus-scan"
```

Comment:
`gdown` currently does not account for the virus scan issue. The branch has a PR  attached that fixes this wkentaro/gdown#308.